libs/utils/src/types.ts

```diff
@@ -3,51 +3,84 @@ import {
-  TallySourceMachineType
+export const ReportSourceMachineTypeSchema = z.nativeEnum(
+  ReportSourceMachineType
 );
 
 export type BallotCountDetails = [precinct: number, absentee: number];
 
-export interface PrecinctScannerCardTally {
-  readonly tallyMachineType: TallySourceMachineType.PRECINCT_SCANNER;
+export const BallotCountDetailsSchema: z.ZodSchema<BallotCountDetails> =
+  z.tuple([z.number(), z.number()]);
+
+export interface PrecinctScannerCardReportBase {
+  readonly tallyMachineType: ReportSourceMachineType.PRECINCT_SCANNER;
   readonly machineId: string;
+  readonly isLiveMode: boolean;
+  readonly precinctSelection: PrecinctSelection;
+  readonly totalBallotsScanned: number;
   readonly timeSaved: number;
   readonly timePollsTransitioned: number;
-  readonly totalBallotsScanned: number;
-  readonly isLiveMode: boolean;
-  readonly pollsTransition: PollsTransition;
+}
+
+export const PrecinctScannerCardReportBaseSchema = z.object({
+  tallyMachineType: z.literal(ReportSourceMachineType.PRECINCT_SCANNER),
+  machineId: MachineId,
+  isLiveMode: z.boolean(),
+  precinctSelection: PrecinctSelectionSchema,
+  totalBallotsScanned: z.number(),
+  timeSaved: z.number(),
+  timePollsTransitioned: z.number(),
+});
+
+export interface PrecinctScannerCardTallyReport
+  extends PrecinctScannerCardReportBase {
+  readonly pollsTransition: StandardPollsTransition;
   readonly ballotCounts: Dictionary<BallotCountDetails>;
-  readonly precinctSelection: PrecinctSelection;
   readonly talliesByPrecinct?: Dictionary<CompressedTally>;
   readonly tally: CompressedTally;
 }
-export const PrecinctScannerCardTallySchema: z.ZodSchema<PrecinctScannerCardTally> =
-  z.object({
-    tallyMachineType: z.literal(TallySourceMachineType.PRECINCT_SCANNER),
+
+export const PrecinctScannerCardTallyReportSchema: z.ZodSchema<PrecinctScannerCardTallyReport> =
+  PrecinctScannerCardReportBaseSchema.extend({
+    pollsTransition: StandardPollsTransitionSchema,
     tally: CompressedTallySchema,
-    machineId: MachineId,
-    timeSaved: z.number(),
-    timePollsTransitioned: z.number(),
-    totalBallotsScanned: z.number(),
-    isLiveMode: z.boolean(),
-    pollsTransition: PollsTransitionSchema,
-    precinctSelection: PrecinctSelectionSchema,
     talliesByPrecinct: z.object({}).catchall(CompressedTallySchema).optional(),
-    ballotCounts: z.object({}).catchall(z.tuple([z.number(), z.number()])),
+    ballotCounts: z.object({}).catchall(BallotCountDetailsSchema),
+  });
+
+export interface PrecinctScannerCardBallotCountReport
+  extends PrecinctScannerCardReportBase {
+  pollsTransition: PollsSuspensionTransition;
+}
+
+export const PrecinctScannerCardBallotCountReportSchema: z.ZodSchema<PrecinctScannerCardBallotCountReport> =
+  PrecinctScannerCardReportBaseSchema.extend({
+    pollsTransition: PollsSuspensionTransitionSchema,
   });
 
+export type PrecinctScannerCardReport =
+  | PrecinctScannerCardBallotCountReport
+  | PrecinctScannerCardTallyReport;
```

Comment:
It wasn't done to avoid conflicts, just to indicate that this data format is specifically for the saving to the card. For example, the `CompressedTally` was specifically made for the purpose of saving to a card compactly.

That's the background, but I agree it's unnecessary. I'll just remove the card and make it less wordy.

Reply:
After looking at it, I wanted to preserve some naming distinction between the report itself and the _report data_ we save to the card, so removing `Card` from these names. And also `Precinct`, since it's unnecessary and it already seems like we anticipate the possibility of some other report source. But adding `Data`.

Reply:
So `PrecinctScannerCardTallyReport` -> `ScannerCardTallyReportData`

Reply:
Makes sense! I like the new naming schema

Correction to the above comment: I think you've ultimately gone with `ScannerTallyReportData` (not `ScannerCardTallyReportData`)

Reply:
Yup!